dags/synaptor_dags.py

```diff
@@ -109,3 +109,243 @@
     >> generate_self_destruct
     >> wait_self_destruct
 )
+
+
+# =========================================
+# DB segmentation
+# "run synaptor db segmentation"
+#
+# This performs the "merge_ccs" task above in separate extra steps (in parallel)
+#
+# I will also soon add some database management commands (creating indexes)
+
```

Comment:
Do you need synaptor_file_seg and synaptor_db_seg in a single run? Just wondering if you can merge them to a single data based on options in the input parameters

Reply:
No. I could merge all of these dags using the current configuration file if I wanted to. I wrote it out this way so that new users could easily read the different ways of running the code. Making an explicit choice about the backend is also a reminder to make sure that the database is set up and connected to the cluster. Adding some if statements certainly isn't difficult for this though. What do you think?

Reply:
No problem, I do not know which approach is more useful in the long run, let's ship this and see how it will evolve